server/KonomiTV-Service.py

```
@@ -254,6 +256,132 @@ def install(
+
+        # すでに完全修飾（DOMAIN\user）されているが、.\\ で始まらない場合はそのまま返す
+        if '\\' in account_name and not account_name.startswith('.\\'):
+            # ドメイン（またはコンピュータ名）とユーザー名に分解
+            input_domain, input_user = account_name.split('\\', 1)
+            
+            # 入力されたドメイン部分が "." の場合はローカル SAM に置き換える
+            search_server = None if input_domain == '.' else input_domain
+            
+            try:
+                # 指定されたサーバーに対して直接照会
+                sid, found_domain, account_type = win32security.LookupAccountName(search_server, input_user)
```

Ruff (0.16.3)

[warning] Unpacked variable `sid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)